### ISSUE TITLE
Patients confuse teleconsultation experience with treatment efficacy

### ISSUE THREAD

**Issue by vibhabhirud** [Intelehealth/Android-Mobile-Client#609]
On the patient satisfaction survey screen, the language of the question should change to "How was your experience with the teleconsultation?"

**Comment by nehagoel2** [comment]
@vibha- I disagree, we aren't collecting their feedback about just the experience with the consultation. Its important to take their feedback about their percieved treatment efficiency. Let's not lower the bar for how we measure patient satisfaction.
